src/test/resources/data/AutomatedSessionRemindersTest.json

```diff
@@ -4,7 +4,7 @@
   	  "googleId": "AutSessRem.instructor",
   	  "name": "Test Ins for Aut Sessions Reminder",
   	  "isInstructor": true,
-  	  "email": "${support.email}",
+  	  "email": "${value.to.be.overridden}",
```

Comment:
I think we can use a more descriptive value that states that the value will be overridden in the test file. This will help a new developer to quickly identify the location where the value is obtained.

Not sure if this is needed. Expected use case is when we generate these values in different places and not just the test file. What do you think?